tests/functional/builtins/codegen/test_abi_decode.py

```diff
@@ -8,6 +8,8 @@
 
 TEST_ADDR = "0x" + b"".join(chr(i).encode("utf-8") for i in range(20)).hex()
 
+MSG_CALL_ABI_OVERHEAD_SZ = 4 + 2 * 32
```

Comment:
shorter - `BUFFER_OVERHEAD`

Reply:
resolved in https://github.com/vyperlang/vyper/pull/4096/commits/3925c8b3c0bffb19bf766e8114d19a8c5cef5d7f